### PR TITLE
adds layers page

### PR DESCRIPTION
**Pull request by ykethan**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  layers: {
+   "@aws-lambda-powertools/logger":
+      "arn:aws:lambda:us-east-1:094274105915:layer:AWSLambdaPowertoolsTypeScriptV2:12",
+  },
+});
+```
+
+The Lambda layer is represented by an object of key/value pairs where the key is the module name that is exported from your layer and the value is the ARN of the layer. The key (module name) is used to externalize the module dependency so it doesn't get bundled with your lambda function. A maximum of 5 layers can be attached to a function, and they must be in the same region as the function.
+
+then use the locally installed module in the function handler:
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx:48]:
```suggestion
Then use the locally installed module in the function handler:
```

File: src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```ts title="amplify/functions/my-function/handler.ts"
+import { Logger } from "@aws-lambda-powertools/logger";
+
+const logger = new Logger({ serviceName: "serverlessAirline" });
+
+export const handler = async (): Promise<void> => {
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx:54]:
```suggestion
export const handler = async (): Promise<void> => {
```

can we show this with one of the generic `Handler` lambda types?

Comment (ykethan) [reply src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx]: updated the snippet

File: next-env.d.ts [hunk 3]
```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/pages/building-your-application/configuring/typescript for more information.
```
Comment (josefaidt) [inline next-env.d.ts:5]: can we discard the changes to this file?

**Review by josefaidt — APPROVED** [review]

**Review by esauerbo — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+Amplify offers the ability to add layers to your Functions which contain your library dependencies. To get started, specify the `layers` property in `defineFunction`:
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx:32]: lowercase f on functions

**Review by aspittel — APPROVED** [review]

**Review by swaminator — COMMENTED** [review]
We need to tell customers explicitly that if they want to set up a layer, they should first do that and reference it in their Amplify project.

**Pull request merged**